content/en/docs/components/training/overview.md

```diff
@@ -1,79 +1,89 @@
 +++
 title = "Overview"
-description = "An overview of the Training Operator"
+description = "An overview of the Kubeflow Training"
 weight = 10
 +++
 
-{{% stable-status %}}
+{{% alert title="Note" color="dark" %}}
+The Kubeflow Training V2 is currently in <strong>alpha</strong> status, and APIs may change.
```

Comment:
This is missing what Kubeflow Training is - an API? Something else? It sounds strange to say "The Kubeflow Training."

Reply:
Let me say Kubeflow Training project, WDYT @vsoch ?